doc/lsst.afw.table/index.rst

```diff
@@ -23,33 +23,19 @@ Contributing
 ``lsst.afw.table`` is developed at https://github.com/lsst/afw.
 You can find Jira issues for this module under the `afw <https://jira.lsstcorp.org/issues/?jql=project%20%3D%20DM%20AND%20component%20%3D%20afw>`_ component.
 
-.. toctree::
-   :maxdepth: 2
-
-   design-notes
-
 .. _lsst.afw.table-pyapi:
 
 Python API reference
 ====================
 
 .. automodapi:: lsst.afw.table
+   :no-main-docstr:
 
-.. automodapi:: lsst.afw.table.base
-.. automodapi:: lsst.afw.table.ampInfo
-.. automodapi:: lsst.afw.table.exposure
 .. automodapi:: lsst.afw.table.io
-.. automodapi:: lsst.afw.table.match
-.. automodapi:: lsst.afw.table.aggregates
-.. automodapi:: lsst.afw.table.aliasMap
-.. automodapi:: lsst.afw.table.arrays
-.. automodapi:: lsst.afw.table.baseColumnView
-.. automodapi:: lsst.afw.table.schema
+   :no-main-docstr:
+
 .. automodapi:: lsst.afw.table.schemaMapper
```

Comment:
Oops, it didn't occur to me that #476 might affect the docs. `lsst.afw.table.schemaMapper` no longer exists; `afw.table` is one monolithic package now.

Reply:
@kfindeisen Thanks for the tip on that